packages/openneuro-components/src/search-page/SearchResultItem.tsx

```diff
@@ -164,7 +165,7 @@ export const SearchResultItem = ({
   const size = (
     <span className="result-summary-meta">
       <strong>Size: </strong>
-      <span>{bytes(summary?.size)}</span>
+      <span>{bytes(node?.latestSnapshot?.size)}</span>
```

Comment:
For the curious reader:
> bytes(x?.bad)
null

Reply:
Good `Maybe` monad.

Reply:
Nice, so that's mostly fine (it will render "Size: ").

Reply:
Would this be a reasonable failure mode?

```suggestion
      <span>{bytes(node?.latestSnapshot?.size) || "unknown" }</span>
```